Jenkinsfile

```diff
@@ -117,28 +75,23 @@ pipeline {
           agent {
             docker {
               image "${nodeImage}"
-              label nodeName
               args '-u root -v /etc/pki:/certs'
             }
           }
           steps {
-            sh 'make install'
-            sh 'make developmentTests'
+            runDevelopmentTests()
           }
         }
-
         stage('Test Production and Zip Production') {
           agent {
             docker {
               image "${nodeImage}"
-              label nodeName
               args '-u root -v /etc/pki:/certs'
             }
           }
           steps {
             // Testing
```

Comment:
can we remove this comment, seems unneeded to me, the method name is descriptive enough

Reply:
You pointed at the `label nodeName`? Do you want me to remove this?

Reply:
ohhhh no not sure what happened here. just ignore me for now. This can go ahead